integration-tests/src/test/java/org/apache/fineract/integrationtests/common/loans/LoanTransactionHelper.java

```diff
@@ -2792,6 +2793,23 @@ public PostLoansLoanIdResponse disburseLoan(Long loanId, PostLoansLoanIdRequest
         return Calls.ok(FineractClientHelper.getFineractClient().loans.stateTransitions(loanId, request, "disburse"));
     }
 
+    /**
```

Comment:
Use this instead: org.apache.fineract.integrationtests.common.loans.LoanTransactionHelper#disburseLoan(java.lang.Long, org.apache.fineract.client.models.PostLoansLoanIdRequest)